### PR TITLE
feat(pam/model): Add IsAuthenticated cancelled event and forward it to the model

### PR DESCRIPTION
**Pull request by 3v1n0**
In this way we can just cancel an auth request from gdm side without having to change stage

UDENG-2565

### CODE REVIEW

File: pam/internal/gdm/gdm.proto [hunk 1]
```diff
@@ -106,7 +106,9 @@ enum EventType {
     // EventType_isAuthenticatedRequested is an authentication request EventType.
     isAuthenticatedRequested = 10;
     // EventType_stageChanged is stage changed EventType.
-    stageChanged = 11;
+    stageChanged = 12;
+    // EventType_isAuthenticationCancelled is authentication cancellation EventType.
+    isAuthenticatedCancelled = 11;
```
Comment (denisonbarbosa) [inline pam/internal/gdm/gdm.proto:111]: 10 -> 12 -> 11, ouch...

Can we keep the numerical order here?

**Review by denisonbarbosa — CHANGES_REQUESTED** [review]
Looks good to me! Only one nitpick...

File: pam/internal/gdm/gdm.proto (hunk 1 above)
Comment (3v1n0) [reply pam/internal/gdm/gdm.proto:111]: Oh, this meant to be before stageChanged, reason why the order wasn't respected.

**Review by denisonbarbosa — APPROVED** [review]
Nice work!

**Comment by denisonbarbosa** [comment]
@3v1n0, can you rebase this one so we can merge it?

**Pull request merged**